src/test/java/net/sf/jabref/model/entry/EntryUtilTest.java

```diff
@@ -23,4 +25,12 @@ public void getSeparatedKeywords() {
         Assert.assertArrayEquals(expected, separatedKeywords.toArray());
     }
 
+    @Test
+    public void testQuote() {
```

Comment:
I'll have a look at it. There will be a few more tests as well as there
will be more than one quote method (these tests were only moved from
StringUtil).